src/common/TransferAmountCell.tsx

```diff
@@ -1,34 +1,36 @@
 import { Transaction } from "@logion/client/dist/TransactionClient.js";
-import { Numbers } from "@logion/node-api";
+import { Lgnt, LgntFormatter } from "@logion/node-api";
 
 import { GREEN, RED } from "./ColorTheme";
 import Icon from "./Icon";
 import { Cell } from "./Table";
 
 import './TransferAmountCell.css';
-import { toPrefixedLgnt } from "./AmountCell";
+import { toUnit } from "./AmountFormat";
+import { toFeesClass } from "@logion/client";
 
 export interface Props {
-    amount: Numbers.PrefixedNumber | null,
+    amount: Lgnt | null,
 }
 
 export default function TransferAmountCell(props: Props) {
+    const formatter = new LgntFormatter(2, toUnit(props.amount || Lgnt.zero()));
     if(props.amount === null) {
         return <Cell content="-" align="center" />;
-    } else if(props.amount.isNegative()) {
+    } else if(props.amount.canonical < 0n) {
         return (
             <div className="TransferAmountCell">
                 <Icon icon={{id: 'send_red'}} />
                 <span className="number" style={{color: RED}}>
-                    { props.amount.coefficient.toFixedPrecision(2) + " " + props.amount.prefix.symbol }
+                    { formatter.format(props.amount) }
```

Comment:
Explicit the (default) number of decimals?

Reply:
Not sure to understand what you mean - the number of decimals is given as a parameter of the formatter, 8 lines above this one. As it does not look critical, I will temporarily leave it unchanged.

Reply:
Indeed, my bad!